packages/admin-panel/src/authentication/selectors.js

```diff
@@ -17,3 +17,12 @@ export const getIsUserAuthenticated = state => !!getAuthenticationState(state).u
 
 // User details
 export const getUser = state => getAuthenticationState(state).user || {}; // If null, return empty object
+
+// Assume that if a user has any BES Admin access, they are an internal user, to avoid having to check permissions for every country
+export const getHasBESAdminPanelAccess = state => {
+  const user = getUser(state);
+  if (!user || !user.accessPolicy) return false;
+  return Object.keys(user.accessPolicy).some(countryCode =>
+    user.accessPolicy[countryCode].some(permissionGroupName => permissionGroupName === 'BES Admin'),
+  );
+};
```

Comment:
I tossed up whether to add this in the FE or BE, and went with FE because the login gets handled by server-boilerplate and I didn't want to over-engineer it, and the FE has has access to the user's access policy already anyway.